doc/guides/CustomStartupGuide.md

```diff
@@ -452,8 +455,85 @@ Follow these instructions to update devices.
+>**NOTE:** Ensure that the cameras are all installed and configured before attempting discovery.    
+
+> For `Netscan`, there is a one line command to determine the `DiscoverySubnets` of your current machine:
+> ```shell
+> ip -4 -o route list scope link | sed -En "s/ dev ($(find /sys/class/net -mindepth 1 -maxdepth 2 -not -lname '*devices/virtual*' -execdir grep -q 'up' "{}/operstate" \; -printf '%f\n' | paste -sd\| -)).+//p" | grep -v "169.254.0.0/16" | sort -u | paste -sd, -
+> ```
+> Example Output: `192.168.1.0/24`
 
- Map credentials using the `map-credentials.sh` script.  
+>**NOTE:** Alternatively, for `netscan` you can set the `DiscoverySubnets` automatically _after_ the service has been deployed by running the [bin/configure-subnets.sh](../utility-scripts.md#configure-subnetssh) script
+
+#### 1. Discovery Configuration
+
+> _See the [Auto Discovery Configuration Guide](../auto-discovery.md#Configuration-Guide)  for full details_
+>
+<details>
+<summary><strong>via configuration.toml</strong></summary>
+
+Define the following configurations in [cmd/res/configuration.toml](../../cmd/res/configuration.toml) for auto-discovery mechanism:
+
+```toml
+[Device]
+    [Device.Discovery]
+    Enabled = true    # enable device discovery
+    Interval = "1h"   # set to desired interval
+
+# Custom configs
+[AppCustom]
+# The target ethernet interface for multicast discovering
+DiscoveryEthernetInterface = "eth0"
+# The Secret Path of the default credentials to use for devices
+DefaultSecretPath = "credentials001"
+# Select which discovery mechanism(s) to use
+DiscoveryMode = "both" # netscan, multicast, or both
+# List of IPv4 subnets to perform netscan discovery on, in CIDR format (X.X.X.X/Y)
+# separated by commas ex: "192.168.1.0/24,10.0.0.0/24"
+DiscoverySubnets = "192.168.1.0/24" # Fill in with your actual subnet(s)
+```
+</details>
+
+<details>
+<summary><strong>via Docker / Env Vars</strong></summary>
+
+Define the following environment variables in `docker-compose.yaml`:
+```yaml
+device-onvif-camera:
+  environment:
+    DEVICE_DISCOVERY_ENABLED: "true"  # enable device discovery
+    DEVICE_DISCOVERY_INTERVAL: "1h"   # set to desired interval
+
+    # The target ethernet interface for multicast discovering
+    APPCUSTOM_DISCOVERYETHERNETINTERFACE: "eth0"
+    # The Secret Path of the default credentials to use for devices
+    APPCUSTOM_DEFAULTSECRETPATH: "credentials001"
+    # Select which discovery mechanism(s) to use
+    APPCUSTOM_DISCOVERYMODE: "both" # netscan, multicast, or both
+    # List of IPv4 subnets to perform netscan discovery on, in CIDR format (X.X.X.X/Y)
+    # separated by commas ex: "192.168.1.0/24,10.0.0.0/24"
+    APPCUSTOM_DISCOVERYSUBNETS: "192.168.1.0/24" # Fill in with your actual subnet(s)
+```
+</details>
+
+<br/>  
+
+Device discovery is triggered by the device SDK. Once the device service starts, it will discover the Onvif camera(s) at the specified interval.
```

Comment:
"SDK" seems odd... what is this? I get Software Development Kit, but can we up level it to something more meaningful?